test/actions/files.test.js

```diff
@@ -39,7 +39,7 @@ describe('Actions.Files', () => {
         const formBoundary = imageFormData.getBoundary();
 
         const fileUploadResp = await basicClient.
-            uploadFile(basicTeam.id, basicChannel.id, clientId, imageFormData, formBoundary);
+            uploadFile(basicTeam.id, basicChannel.id, imageFormData, formBoundary);
```

Comment:
Mistake on my part, I'll add a unit test for upload and update the create post action test.